src/components/Anatomy.jsx

```diff
@@ -172,4 +177,240 @@ export const Anatomy = () => {
+  };
+  const checkDistanceOnDragSpine = (l, dl, w, dw) => {
+    checkDistanceOnDrag(l, dl, w, dw, spineSkeleton.targetPosition, spineSkeleton.distanceThreshold, setReachedTargetSpine)
+  };
+  const checkDistanceOnDragRibs = (l, dl, w, dw) => {
+    checkDistanceOnDrag(l, dl, w, dw, ribsSkeleton.targetPosition, ribsSkeleton.distanceThreshold, setReachedTargetRibs)
+  };
+
+
+  const Jaw = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={jawSkeleton} checkDistanceOnDrag={checkDistanceOnDragJaw} draggable={draggable} rotation={[2.2, 0, 0]} />
+  };
+  const Skull = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={skullSkeleton} checkDistanceOnDrag={checkDistanceOnDragSkull} draggable={draggable} rotation={[0, 0, 0]} />
+  };
+  const UpperLegL = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={upperLegLSkeleton} checkDistanceOnDrag={checkDistanceOnDragUpperLegL} draggable={draggable} rotation={[0, 0, 0]} />
+  };
+  const UpperLegR = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={upperLegRSkeleton} checkDistanceOnDrag={checkDistanceOnDragUpperLegR} draggable={draggable} rotation={[0, 0, 0]} />
+  };
+  const FibulaR = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={fibulaRSkeleton} checkDistanceOnDrag={checkDistanceOnDragFibulaR} draggable={draggable} rotation={[3.1, 0, 0]} />
+  };
+  const FibulaL = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={fibulaLSkeleton} checkDistanceOnDrag={checkDistanceOnDragFibulaL} draggable={draggable} rotation={[3.1, 0, 0]} />
+  };
+  const LowerLegR = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={lowerLegRSkeleton} checkDistanceOnDrag={checkDistanceOnDragLowerLegR} draggable={draggable} rotation={[3.15, 0, 0]} />
+  };
+  const LowerLegL = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={lowerLegLSkeleton} checkDistanceOnDrag={checkDistanceOnDragLowerLegL} draggable={draggable} rotation={[3.15, 0, 0]} />
+  };
+  const LeftArm = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={leftArmSkeleton} checkDistanceOnDrag={checkDistanceOnDragLeftArm} draggable={draggable} rotation={[0.011354048101568781, 0.014001609606136218, -0.6813951991697313]} anchorPosition={[0.3, 0.3, 0.3]} />
+  };
+  const RightArm = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={rightArmSkeleton} checkDistanceOnDrag={checkDistanceOnDragRightArm} draggable={draggable} rotation={[0.09841546379069474, -0.07174505620204256, 0.6956416135179724]} anchorPosition={[-0.8, 0.4, 0.4]} />
+  };
+  const Spine = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={spineSkeleton} checkDistanceOnDrag={checkDistanceOnDragSpine} draggable={draggable} rotation={[-1.5, 0, 0]} />
+  };
+  const Ribs = ({ draggable }) => {
+    return <SkeletonDisplay skeleton={ribsSkeleton} checkDistanceOnDrag={checkDistanceOnDragRibs} draggable={draggable} />
+  };
```

Comment:
You can use a dictionary/object to prevent multiple lines of codes like this. (Maybe at the very least separate it into a different file)